source/includes/api/_dashboard.md

```diff
@@ -281,7 +281,21 @@ curl -X PATCH https://api.resourcewatch.org/v1/dashboards/<id of the dashboard>
 }
 ```
  
+## Delete dashboard
 
+In order to perform this operation, the following conditions must be met:
+
+- the user must be logged in and belong to the same application as the dashboard
+- the user must match one of the following:
+  - have role `ADMIN`
+  - have role `MANAGER` or `USER` and be the dashboard's owner (through the `user-id` field of the dashboard)
```

Comment:
I think, technically, right now, any authenticates user can delete a dashboard. However, this is probably due to an oversight, and I think we should aim to change this so it mimics other resources (datasets, widgets, etc) where admins can delete any, managers their own, and users none